projects/unpaid-work-and-delius/build.gradle.kts

```diff
@@ -20,6 +20,8 @@ dependencies {
     implementation(libs.openfeign)
     implementation(libs.sentry)
     implementation(libs.springdoc)
+    implementation("org.mapstruct:mapstruct:1.5.3.Final")
```

Comment:
Should we move the version up to settings.gradle, in case we want to use it other projects?